#31081: Document that Tor accepts GETCONF with zero or more arguments (#104)
torproject/torspec: control-spec.txt

3.3. GETCONF

>   Request the value of a configuration variable(s).  The syntax is:

Reviewer: ```suggestion
  Request the value of zero or more configuration variables.  The syntax is:
```
